src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -70,5 +70,6 @@ internal enum ConversionKind : byte
         InlineArray, // A conversion from an inline array to Span/ReadOnlySpan
 
         ImplicitSpan, // A conversion between array, (ReadOnly)Span, string - part of the "first-class Span types" feature
```

Comment:
>// A conversion between array, (ReadOnly)Span, string [](http://example.com/codeflow?start=22&length=53)

Consider making the comment for `ImplicitSpan` more specific, to match the comment for `ExplicitSpan`. Perhaps:
```
// A conversion from array to (ReadOnly)Span, or from string or (ReadOnly)Span to ReadOnlySpan
```